suite-native/module-onboarding/src/screens/ReceiveCoinsScreen.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    OnboardingStackParamList,
+    OnboardingStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Box } from '@suite-native/atoms';
+
+import { GetTrezor } from '../components/GetTrezor';
+import { OnboardingScreen } from '../components/OnboardingScreen';
+import { CoinsSvg } from '../components/CoinsSvg';
+
+export const ReceiveCoinsScreen = () => {
```

Comment:
Please try to come up with different name, this is very confusing because we already have screen for Receive :D